src/media/js/routes_api.js

```diff
@@ -17,6 +17,7 @@ define('routes_api', [], function() {
         'feedback': '/api/v2/account/feedback/',
         'fxa-login': '/api/v2/account/fxa-login/',
         'installed': '/api/v2/account/installed/mine/',
+        'langpacks': '/api/v2/langpacks/?cache=1&vary=0',
```

Comment:
gives a `Cache-Control: max-age=180` to the API. I could drop the vary=0, but it'd be useful it we decide to cache this endpoint on the CDN.